src/test/test.cpp

```diff
@@ -636,6 +636,36 @@ recordOrCheckGlobalTestTxMetadata(TransactionMeta const& txMetaIn)
     }
 }
 
+static char const* TESTKEY_PROTOCOL_VERSION = "!cfg protocol version";
+static char const* TESTKEY_RNG_SEED = "!rng seed";
+static char const* TESTKEY_ALL_VERSIONS = "!test all versions";
+static char const* TESTKEY_VERSIONS_TO_TEST = "!versions to test";
+
+template <typename T>
+void
+checkTestKeyVal(std::string const& k, T const& expected, T const& got,
+                std::string const& path)
+{
```

Comment:
standard compliance issue: you're declaring `path` as an `std::string` but you're calling with a `std::filesystem::path` and there is no implicit way to convert a path into a string (for whatever reason clang is happy to do it but not VC++)

Reply:
fixed